packages/fluentui/react-bindings/src/icons/types.ts

```diff
@@ -60,3 +60,13 @@ export type SvgIconCreateFnParams<TProps> = {
   displayName: string;
   handledProps?: (keyof TProps)[];
 };
+
+export type ImgIconProps = { size?: number; sizeModifier?: '1.5x' | '2x' | '3x' | '4x' };
```

Comment:
What props do we actually want to support here? Can we decide on common icon props for both libraries? @dzearing